src/sugar/cli.py

```diff
@@ -155,13 +77,1672 @@ def extract_options_and_cmd_args() -> Tuple[list, list]:
+        ),
+        all: bool = Option(
+            False,
+            help='Use all services for the command.',
+            is_flag=True,
+        ),
+        config_file: str = Option(
+            str(Path(os.getcwd()) / '.sugar.yaml'),
+            help='Specify a custom location for the config file.',
+            is_flag=True,
+        ),
+        cmd: str = Option(
+            None,
+            help='Specify the COMMAND for some docker-compose command.\
+                E.g.: --cmd python -c print(1)',
+        ),
+        verbose: bool = Option(
+            False,
+            '--verbose',
+            is_flag=True,
+            is_eager=True,
+            help='Show the command executed.',
+        ),
+    ):
+        args = ctx.params
+        args['plugin'] = 'ext'
+        args['action'] = 'restart'
+
+        if verbose or state['verbose']:
+            args['verbose'] = True
+
+        cmd_args: List[Any] = state['cmd']
+        opts_args: List[Any] = state['options']
+
+        Sugar(args, options_args=opts_args, cmd_args=cmd_args).run()
+
+    @ext_group.command()
+    def wait(
+        ctx: typer.Context,
+        service_group: str = Option(
+            None,
+            '--service-group',
+            '--group',
+            help='Specify the group name of the services you want to use',
+        ),
+        services: str = Option(
+            None,
+            help="Set the services for the container call.\
+                Use comma to separate the services's name",
+        ),
+        service: str = Option(
+            None, help='Set the service for the container call.'
+        ),
+        options: str = Option(
+            None,
+            help='Specify the options for docker-compose command.\
+                E.g.: --options -d',
+        ),
+        all: bool = Option(
+            False,
+            help='Use all services for the command.',
+            is_flag=True,
+        ),
+        config_file: str = Option(
+            str(Path(os.getcwd()) / '.sugar.yaml'),
+            help='Specify a custom location for the config file.',
+            is_flag=True,
+        ),
+        cmd: str = Option(
+            None,
+            help='Specify the COMMAND for some docker-compose command.\
+                E.g.: --cmd python -c print(1)',
+        ),
+        verbose: bool = Option(
+            False,
+            '--verbose',
+            is_flag=True,
+            is_eager=True,
+            help='Show the command executed.',
+        ),
+    ):
+        args = ctx.params
+        args['plugin'] = 'ext'
+        args['action'] = 'wait'
+
+        if verbose or state['verbose']:
+            args['verbose'] = True
+
+        cmd_args: List[Any] = state['cmd']
+        opts_args: List[Any] = state['options']
+
+        Sugar(args, options_args=opts_args, cmd_args=cmd_args).run()
+
+    # -- Stats Commands --
+
+    @stats_group.command()
+    def plot():
+        print('Plot command')
```

Comment:
it seems that this function is incomplete.

Reply:
Sorry haha, I forgot this one xD